docs/includes/file-get-started-powerpoint.md

```diff
@@ -230,7 +230,7 @@ In this article, you'll walk through the process of building a PowerPoint add-in
     </div>
     ```
 
-4. Open the file **app.js** in the root of the project to specify the script for the add-in. Replace the entire contents with the following code and save the file.
+4. Open the file **index.js** in the root of the project to specify the script for the add-in. Replace the entire contents with the following code and save the file.
```

Comment:
I just changed this for OneNote. Is this in the root folder or src/index.js?

Reply:
under src. I'll update it. Thx.